src/components/Modals/CourseMenu.vue

```diff
@@ -191,7 +191,7 @@ export default {
     justify-content: space-between;
     padding: 0.5rem 1rem;
     position: relative;
-
+    cursor: pointer;
```

Comment:
@willespencer so typically when a style is repeated, I have the opinion that it's better to just put that into its own class like "cursor-pointer" than to put it into every class. Wdyt?

Reply:
yeah, i was thinking about doing that initially and then i realized that i was adding the class pointer everywhere on the template. so, i kinda looked at the pre-existing classes, and concluded that adding the 1-liner `cursor: pointer;` to the css seemed more concise.

Reply:
Yeah what Han said is def good practice, but I also agree with Hahnbee that this is more concise in this specific case, plus we don't really do that anywhere in our codebase as of now 😕. So leaning towards leaving it as is.